<launch>
  <!-- Load the turtlebot3_empty_world.launch file -->
  <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_empty_world.launch" />

  <!-- Map Gazebo's robot pose data to match Vicon's robot pose data format  -->
  <node pkg="ros_sgp_ipp" name="gazebo_pose_publisher" type="gazebo_pose_publisher.py" output="screen" />

  <!-- Load the rssi publisher node -->
  <node pkg="ros_sgp_ipp" name="rssi_publisher" type="rssi_publisher.py" output="screen" />

  <!-- Load the trajectory planner and online IPP nodes -->
  <node pkg="ros_sgp_ipp" name="trajectory_planner" type="trajectory_planner.py" output="screen" />

  <!-- Load the online IPP node -->
  <node pkg="ros_sgp_ipp" name="onlineIPP" type="onlineIPP.py" output="screen" />
</launch>